src/obfl-specification.html

```diff
@@ -1225,6 +1225,26 @@ <h4 id="L1023" class="include">block</h4>
             left at the top of a page.</dd>
           <dd>An integer in the range [1, 9].</dd>
           <dd>Default is 1.</dd>
+        <dt>display-when [optional]</dt>
+          <dd>An condition that determines whether the contents of the block are
+            rendered.</dd>
+          <dd>Default is true.</dd>
+          <dd class="markup note">All markers in this block and its descendant
+            elements will be processed, regardless of the outcome of the
+            condition. So it is possible to use a marker's value even when the
+            block is not rendered.</dd>
+          <dd class="markup note">If this display-when condition is present, all
+            identifier attributes, anchors and corresponding collection items in
+            this block and all its descendant elements will be disregarded.</dd>
```

Comment:
Maybe it's better to disallow these elements when display-when is present? I don't see a reason to allow them if they are disregarded anyway, because it's something that the OBFL author can easily fix.

Reply:
>Maybe it's better to disallow these elements when display-when is present?
That is a more strict approach, but the drawback is that you need to explicitly allow these elements if you intend to make use of them in the future.
The reason why I put it like this is because we have a similar sentence [here](https://mtmse.github.io/obfl/obfl-specification.html#thead). It reads there "Note that the identifier attribute will be disregarded for all descendant elements of thead."

Reply:
Well, it's really easy to extend OBFL later if needed. I'm just concerned about allowing certain elements and then just ignoring them. Allowing them will result in certain expectations from the user.

Either we allow and (intend to) process them, or we don't allow.

Reply:
By the way, good that you noticed that sentence under **thead**. Maybe that should be changed as well.

Reply:
I have replaced "disregarded" with "forbidden" everywhere.